<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detial</class>
 <widget class="QWidget" name="detial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>工序：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a02_order">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="table_02">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>产品名</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>工艺数</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>批次数</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>在制数</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="out">
          <property name="minimumSize">
           <size>
            <width>138</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>产品：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a02_name">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="table_03">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>生产工艺</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>产品批次</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>在制数</string>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="out_2">
          <property name="minimumSize">
           <size>
            <width>138</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close">
          <property name="minimumSize">
           <size>
            <width>138</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
